src/client/dfuse/ops/lookup.c

```diff
@@ -267,20 +273,18 @@ check_for_uns_ep(struct dfuse_projection_info *fs_handle,
 		rc = dfs_mount(dfp->dfp_poh, dfs->dfs_coh, O_RDWR,
 			       &dfs->dfs_ns);
 		if (rc) {
-			DFUSE_LOG_ERROR("dfs_mount failed (%d)", rc);
+			DFUSE_LOG_ERROR("dfs_mount() failed: (%s)",
+					strerror(rc));
 			D_GOTO(out_cont, ret = rc);
 		}
 		new_cont = true;
```

Comment:
did you not move this to line #260 ??

Reply:
Yes, this one is redundant now.  I can fix up in a later PR.